### DISCUSSION TITLE
Adding on_click events?

### DISCUSSION THREAD

**Discussion by alexvoronov** [cbouy/mols2grid#2] [Q&A] [answered]
Hi!

The grid of molecules looks really nice :)

I wonder if there is any way to know if a molecule was clicked on? Maybe there is some way to add `on_click`-events to each molecule/grid-cell? In the end, I'm looking for ways to add a new column to the molecules dataframe with the value True if the molecule was clicked.

**Comment by cbouy** [comment, marked as answer]
There's currently no way to do this for now but I'll add it to the [upcoming features](https://github.com/cbouy/mols2grid/discussions/5) post!

**Reply by alexvoronov** [reply]
Thanks, looking forward to it!

**Reply by cbouy** [reply]
@alexvoronov Molecule selection has been added in [v0.0.3](https://github.com/cbouy/mols2grid/releases/tag/v0.0.3)
See the [demo](https://nbviewer.jupyter.org/github/cbouy/mols2grid/blob/master/demo.ipynb) notebook for an example

**Reply by alexvoronov** [reply]
Great news, thanks!
